tests/many_tables/conf/diff_config.toml

```diff
@@ -0,0 +1,40 @@
+# diff Configuration.
+
+log-level = "info"
+
+chunk-size = 10
+
+check-thread-count = 4
+
+sample-percent = 100
+
+use-rowid = false
+
+use-checksum = true
+
+fix-sql-file = "fix.sql"
+
+# tables need to check.
+[[check-tables]]
+schema = "many_tables_db"
+tables = ["~.*"]
+
+[[source-db]]
+host = "127.0.0.1"
+port = 3306
+user = "root"
+password = "123456"
+instance-id = "source-1"
+
+[[source-db]]
+host = "127.0.0.1"
+port = 3307
+user = "root"
+password = "123456"
+instance-id = "source-2"
```

Comment:
Can we remove these lines?

Reply:
done in https://github.com/pingcap/dm/pull/1301/commits/a8c66301229e028dbd952e76848338c05a8e96c9